tests/failpoints/cases/test_disk_full.rs

```diff
@@ -0,0 +1,257 @@
+// Copyright 2021 TiKV Project Authors. Licensed under Apache-2.0.
+
+use kvproto::raft_cmdpb::*;
+use raftstore::store::msg::*;
+use std::time::Duration;
+use std::time::*;
+use test_raftstore::*;
+use tikv_util::config::*;
+/*
```

Comment:
How about remove this comment and comment on every case itself?

Reply:
ok, I'll change

Reply:
done